src/main/java/com/jcabi/github/mock/MkIssueEvents.java

```diff
@@ -0,0 +1,196 @@
+import com.jcabi.log.Logger;
+import com.jcabi.xml.XML;
+import java.io.IOException;
+import javax.validation.constraints.NotNull;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.xembly.Directives;
+
+/**
+ * Mock GitHub issue events.
+ *
+ * @author Chris Rebert (github@chrisrebert.com)
+ * @version $Id$
+ * @since 0.23
+ */
+@Immutable
+@Loggable(Loggable.DEBUG)
+@ToString
+@EqualsAndHashCode(of = { "storage", "self", "coords" })
+final class MkIssueEvents implements IssueEvents {
+    /**
+     * Storage.
+     */
+    private final transient MkStorage storage;
+
+    /**
+     * Login of the user logged in.
+     */
+    private final transient String self;
+
+    /**
+     * Repo name.
+     */
+    private final transient Coordinates coords;
+
+    /**
+     * Public constructor.
+     * @param stg Storage
+     * @param login User to login
+     * @param rep Repo
+     * @throws IOException If there is any I/O problem
+     */
+    MkIssueEvents(
+        @NotNull(message = "stg can't be NULL") final MkStorage stg,
+        @NotNull(message = "login can't be NULL") final String login,
+        @NotNull(message = "rep can't be NULL") final Coordinates rep
+    ) throws IOException {
+        this.storage = stg;
+        this.self = login;
+        this.coords = rep;
+        this.storage.apply(
+            new Directives().xpath(
+                String.format(
+                    "/github/repos/repo[@coords='%s']",
+                    this.coords
+                )
+            ).addIf("issue-events")
+        );
+    }
+
+    @Override
+    @NotNull(message = "repository is never NULL")
+    public Repo repo() {
+        return new MkRepo(this.storage, this.self, this.coords);
+    }
+
+    @Override
+    @NotNull(message = "issue event is never NULL")
+    public Event get(final int number) {
+        return new MkEvent(this.storage, this.self, this.coords, number);
+    }
+
+    @Override
+    @NotNull(message = "iterable is never NULL")
+    public Iterable<Event> iterate() {
+        return new MkIterable<Event>(
+            this.storage,
+            String.format("%s/issue-event", this.xpath()),
+            new MkIterable.Mapping<Event>() {
+                @Override
+                public Event map(final XML xml) {
+                    return MkIssueEvents.this.get(
+                        Integer.parseInt(xml.xpath("number/text()").get(0))
+                    );
+                }
+            }
+        );
+    }
+
+    /**
+     * Creates a new issue event.
+     * This has no equivalent in GitHub's public API, since GitHub generates
+     * events automatically in response to some other API calls.
+     * @param type Type of event
+     * @param issue ID number of issue the event is regarding
+     * @param login Username of actor who caused the event
+     * @param label Name of label that was added or removed if this is a
+     *  labeled/unlabeled event, or null otherwise.
```

Comment:
@cvrebert let's adjust the description here (no more nulls). I suggest a simple `@param label Label added or removed`.